_includes/nav_events.html

```diff
@@ -0,0 +1,9 @@
+<section>
+  <h1>Events</h1>
+  <ul>
+  {% for event_page in site.events %}
+    <li><a href="{{ event_page.url }}">{{ event_page.title }}</a></li>
+  {% endfor %}
+  </ul>
+</section>
+  
```

Comment:
Leerzeile **und** "no newline at end of file"? WTF, wie geht das? Kann sein, dass diese Detei verschiedene Zeilenenden (`CR`, `CRLF`, `LF`) mischt?

Reply:
lonely space